internal/store/driver.go

```diff
@@ -946,8 +946,7 @@ func (d *Driver) calculateHTTPSEdgesFromGateway(edgeMap map[string]ingressv1alph
 								}
 
 								refName := string(backendref.Name)
-								//refNamespace := string(*backendref.Namespace)
-								serviceUID, servicePort, err := d.getEdgeBackendRef(backendref.BackendRef, gtw.Namespace)
+								serviceUID, servicePort, err := d.getEdgeBackendRef(backendref.BackendRef, httproute.Namespace)
 								if err != nil {
 									d.log.Error(err, "could not find port for service", "namespace", gtw.Namespace, "service", refName)
```

Comment:
looks like we would continue despite the error, is this expected?

Reply:
skipped here: https://github.com/ngrok/kubernetes-ingress-controller/pull/356/commits/477310ccd57ad286a38f0ae7bcbdf2766db25fc5